eng/WpfArcadeSdk/tools/ShippingProjects.props

```diff
@@ -176,6 +185,10 @@
  
     <!-- WindowsDesktopSdk overrides Internal, External etc. -->
     <RepoLocation Condition="$(WindowsDesktopSdkProject.Contains('$(NormalizedMSBuildProjectName)'))">WindowsDesktopSdk</RepoLocation>
+
+    <!-- Test package overrides Internal, External etc. -->
+    <RepoLocation Condition="$(InternalTestProjects.Contains('$(NormalizedMSBuildProjectName)'))">InternalTest</RepoLocation>
```

Comment:
This may not be applicable to test, but I've seen bugs where you have something like:
`TestProject1`
`TestProject1ThatShips`
In these scenarios, just matching the project name would match both, which may be undesirable.  I've been adding the separator character to overcome this, so `'$(NormalizedMSBuildProjectName);'` to give an exact match.

Reply:
it might not apply, but that's a good idea anyways, thanks!